### PR TITLE
 fix income type field name and dropdown

### PR DESCRIPTION
**Pull request by lauherkcambria**
#### What tests were added (if relevant), and do all tests (rspec, karma, and integration CALS tests) pass?
- n/a

#### Which User story does this relate to?
- https://www.pivotaltracker.com/story/show/148677527

#### Brief feature description

- dropdown defaults this value to yearly
- Dropdown cannot be null
- field marked as 'Personal Income'
- income number formatting

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
-

#### Code coverage percent diff as a result of this PR?
-na

#### JS errors diff as a result of this PR?
-na

### CODE REVIEW

**Comment by lauherkcambria** [comment]
jenkins, retest this please.

**Comment by anuroop15** [comment]
@lauherkcambria : 

This PR doesn't address below two requirements as per design document attached to story: 
1) Personal Income Input box value will be right aligned with no hint text
2) Text field supports 8 whole numbers

File: app/javascript/rfa_forms/rfa01a_edit_view/employmentCard.jsx [hunk 1]
```diff
@@ -47,16 +50,21 @@ export default class Employment extends React.Component {
...
+              <CurrencyInput className='col-md-12' id='income'
+                value={employmentFields.income}
+                prefix='$'
+                precision='0'
+                onChangeEvent={(event) => this.onEmploymentChange('income', event.target.value)} />
+            </div>
 
             <DropDownField gridClassName='col-md-1' id='income_type'
-              value={getDictionaryId(employmentFields.income_type)}
+              value={employmentFields.income_type.id}
```
Comment (anuroop15) [inline app/javascript/rfa_forms/rfa01a_edit_view/employmentCard.jsx]: @lauherkcambria : 

Design document shows, there should be only two dropdown values, yearly and monthly. there shouldn't be hourly in dropdown.

Comment (lauherkcambria) [reply app/javascript/rfa_forms/rfa01a_edit_view/employmentCard.jsx]: @anuroop15 this is on purpose. back end team has told me not to fix on our end as they are fixing on theirs.

**Comment by lauherkcambria** [comment]
> 
> This PR doesn't address below two requirements as per design document attached to story:
> 
> Personal Income Input box value will be right aligned with no hint text
> Text field supports 8 whole numbers


@anuroop15 are you following the latest design? I personally showed leon AND design team this design adnd was approved by both.

**Comment by anuroop15** [comment]
@lauherkcambria yeah, I have followed the latest design doc. Below was the link
https://osicagov.sharepoint.com/sites/projects/CWS-NS/Licensing/Cambria%20Private%20Documents/00_Design/03_Final%20Deliverables/Design%20Deliverables/Iteration%204.1/Iteration4.1_employmentAnnualIncome_20170824.pdf

File: app/javascript/rfa_forms/rfa01a_edit_view/employmentCard.jsx (hunk 1 above)
Comment (anuroop15) [reply app/javascript/rfa_forms/rfa01a_edit_view/employmentCard.jsx]: Oh!! sounds good

**Comment by lauherkcambria** [comment]
let's double check and review the AC with leon once he is available 😄

File: app/javascript/rfa_forms/rfa01a_edit_view/employmentCard.jsx (hunk 1 above)
Comment (lauherkcambria) [reply app/javascript/rfa_forms/rfa01a_edit_view/employmentCard.jsx]: backend has completed removing the third option.

**Comment by lauherkcambria** [comment]
@anuroop15 just confirmed AC with leon.

**Comment by lauherkcambria** [comment]
good catch @anuroop15 , added interval label 😄

**Comment by anuroop15** [comment]
Great!! Appreciate the quick fixes 😃 .
As per our discussion on alignment of Income is considered. I'm okay if AC was fine with this. 
Approving the PR. 👍

**Pull request merged**